libraries/libfc/include/fc/io/raw.hpp

```diff
@@ -176,13 +176,15 @@ namespace fc {
     } FC_RETHROW_EXCEPTIONS( warn, "fc::array<${type},${length}>", ("type",fc::get_typename<T>::name())("length",N) ) }
 
     template<typename Stream, typename T, size_t N>
+    requires (!std::is_same_v<std::remove_cv_t<T>, char>)
     inline void pack( Stream& s, T (&v)[N]) {
       fc::raw::pack( s, unsigned_int((uint32_t)N) );
       for (uint64_t i = 0; i < N; ++i)
          fc::raw::pack(s, v[i]);
     }
 
     template<typename Stream, typename T, size_t N>
+    requires (!std::is_same_v<std::remove_cv_t<T>, char>)
```

Comment:
Remember this change from https://github.com/AntelopeIO/spring/pull/1526#discussion_r2093475193. Need to pull in raw.hpp to get the variadic pack(), but that causes an ambiguity with c strings.